### PR TITLE
Datagen

### PR DESCRIPTION
**Pull request by bmpooja**

### CODE REVIEW

File: server/model/product.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const productSchema = new mongoose.Schema({
+  name: {
+    type: String,
```
Comment (hewbahrami) [inline server/model/product.js:5]: I don't think you need to define type for each, you can also write it directly like name: String, but both are ok. Just an FYI

File: server/seeder.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const seeder = require('mongoose-seed');
```
Comment (hewbahrami) [inline server/seeder.js:1]: interesting! i didn't know there was a mongoose seeder npm module. i will check this out!

File: server/seeder.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+const db = 'mongodb+srv://admin:admin@cluster0.mdtqx.mongodb.net/mykea?retryWrites=true&w=majority';
+
+var data = [
+  {
```
Comment (hewbahrami) [inline server/seeder.js:7]: so far you have this one data, yes?

Comment (bmpooja) [reply server/seeder.js:7]: yes, I will work on getting more data now.

**Pull request merged**